src/Nethermind/Nethermind.Consensus.AuRa/AuRaBlockProducer.cs

```diff
@@ -129,40 +129,47 @@ protected override Task<Block> SealBlock(Block block, BlockHeader parent, Cancel
             return base.SealBlock(block, parent, token);
         }
         
+        // This is for debugging.
```

Comment:
Leaving this for now as we want to observe the behavior on the node

Reply:
still?

Reply:
Yes lets leave it for longer. Its not hurting anything.